src/TourManager/Repository/Repositories/GenericRepository.cs

```diff
@@ -0,0 +1,59 @@
+﻿using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Threading.Tasks;
+
+namespace Travely.TourManager.Repository.Repositories
```

Comment:
Move this class to some common library because it can be used by any of our services (equipment, activities, etc)